common/SmokeTests/SmokeTest/SmokeTest.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFrameworks>net7.0;net6.0;net461</TargetFrameworks>
-    <LangVersion>latest</LangVersion>
+    <LangVersion>11.0</LangVersion>
```

Comment:
Can't this override be removed?

Reply:
It is in a sample project, so I want to leave it here in case someone just copy it as a whole